Formula/mvtools.rb

```diff
@@ -24,6 +15,9 @@ class Mvtools < Formula
   depends_on "fftw"
   depends_on "vapoursynth"
 
+  # Fixes build issues on arm
+  patch :DATA
```

Comment:
Has this been upstreamed?

Reply:
I created a pull request, but there hasn't been any commit since 2 years and I forked another fork containing an arm patch for Linux that is already half a year old (not sure why this patch wasn't mainstreamed).